openedx/features/course_duration_limits/tests/test_access.py

```diff
@@ -56,36 +56,34 @@ def test_generate_course_expired_message(self, language, offsets):
         def format_date(date):
             return strftime_localized(date, u'%b %-d, %Y')
 
-        patch_lang = patch('openedx.features.course_duration_limits.access.get_language', return_value=language)
```

Comment:
Why did you need to disable language for these factories?

Reply:
This was changed in ironwood. in juniper this is removed. I missed it during conflict resolution